src/main/java/com/takipi/oss/storage/resources/fs/multifetcher/MultiFetcher.java

```diff
@@ -16,31 +16,75 @@
 public class MultiFetcher {
     
     private static final Logger logger = LoggerFactory.getLogger(MultiFetcher.class);
+    private static long maxCacheSize = 33554432;
     
     private final TaskExecutor taskExecutor;
+    private final static S3Cache cache;
+    
+    static {
+        if (maxCacheSize > 0) {
```

Comment:
What does this mean? Is it typo?

Reply:
If the maxCacheSize was set to zero, then use the dummy cache that does nothing.